### PR TITLE
On prem doco

### PR DESCRIPTION
**Pull request by gotsysdba**
Updates for GA 1.0

### CODE REVIEW

**Review by andytael — APPROVED** [review]

**Comment by gotsysdba** [comment]
moved to 132

File: docs-source/spring/content/on-premises/ol8/_index.md [hunk 1]
```diff
@@ -28,6 +28,15 @@ Set the default Python3 to Python 3.9 by running this command:
 sudo alternatives --set python3 /usr/bin/python3.9
 ```
 
+### Install MiniKube
+
+As the `root` user, install minikube:
```
Comment (rhondaday) [inline docs-source/spring/content/on-premises/ol8/_index.md:33]: Change "... install minikube:" to "... install MiniKube""

**Review by rhondaday — COMMENTED** [review]
I had just one comment.

File: docs-source/mbaas/content/on-premises/ol8/_index.md [hunk 2]
```diff
@@ -12,20 +12,29 @@ Read [On-Premises](../index.md) and ensure that your desktop meets the minimum s
...
 Set the default Python3 to Python 3.9:
 
 ```bash
-sudo alternatives --set python3 /usr/bin/python3.9
+alternatives --set python3 /usr/bin/python3.9
+```
+
+### Install MiniKube
+
+As the `root` user, install minikube:
```
Comment (rhondaday) [inline docs-source/mbaas/content/on-premises/ol8/_index.md:33]: Change "minikube" to "MiniKube".

File: docs-source/spring/content/on-premises/ol8/_index.md [hunk 3]
```diff
@@ -45,26 +54,19 @@ The _Desktop_ installation provisions an Oracle database into the Kubernetes clu
...
+3. Pull the ORDS image. For example:
 
-Install and start Minikube by running these commands as the `root` user:
+   `podman pull container-registry.oracle.com/database/ords:21.4.2-gh`
 
-```bash
-curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
-install minikube-linux-amd64 /usr/local/bin/minikube
-```
+### Start Minikube
```
Comment (rhondaday) [inline docs-source/spring/content/on-premises/ol8/_index.md:69]: "Minikube" should be "MiniKube"

File: docs-source/mbaas/content/on-premises/ol8/_index.md [hunk 4]
```diff
@@ -12,20 +12,29 @@ Read [On-Premises](../index.md) and ensure that your desktop meets the minimum s
...
 ```
 
 Set the default Python3 to Python 3.9:
 
 ```bash
-sudo alternatives --set python3 /usr/bin/python3.9
+alternatives --set python3 /usr/bin/python3.9
+```
+
+### Install MiniKube
```
Comment (rhondaday) [inline docs-source/mbaas/content/on-premises/ol8/_index.md:31]: "MiniKube" should be "Minikube"

File: docs-source/spring/content/on-premises/ol8/_index.md [hunk 5]
```diff
@@ -28,6 +28,15 @@ Set the default Python3 to Python 3.9 by running this command:
 sudo alternatives --set python3 /usr/bin/python3.9
 ```
 
+### Install MiniKube
```
Comment (rhondaday) [inline docs-source/spring/content/on-premises/ol8/_index.md:31]: "MiniKube" should be "Minikube"

**Review by rhondaday — COMMENTED** [review]
Found a few more.

**Pull request closed without merging**